charts/opentelemetry-collector/UPGRADING.md

```diff
@@ -1,4 +1,18 @@
 # Upgrade guidelines
+## 0.22.1 to 0.23.0
```

Comment:
Looks like the charts version needs bumped to 0.24.0 instead, as another PR got merged before this.  Then this doc will need updated and examples will need to be regenerated.

Reply:
yup just noticed, bumped and reran the make generate